srcpkgs/neovim/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'neovim'
 pkgname=neovim
-version=0.7.2
+version=0.8.0
 revision=1
 build_style=cmake
 build_helper="qemu"
 configure_args="-DCMAKE_BUILD_TYPE=Release -DCOMPILE_LUA=OFF"
 hostmakedepends="pkg-config gettext gperf LuaJIT lua51-lpeg lua51-mpack"
 makedepends="libtermkey-devel libuv-devel libvterm-devel msgpack-devel
  LuaJIT-devel libluv-devel tree-sitter-devel"
-depends="libvterm>=0.1.0"
+depends="libvterm>=0.3"
```

Comment:
this dependency is automatically detected, so instead you can just remove this line and change version in common/shlibs to 0.3

Reply:
As per my understanding, one should update the common/shlibs entry only if a different version of the shlib is provided. Otherwise it should stay with the oldest version of the package the provides the shlib. Is that not the case?

Reply:
You can also use it to set the lowest version of a library that can be used. xbps-src scans the package for shared libraries, looks at the entry in common/shlibs and creates a dependency in the form of $pkgname>=${version}_${revision} where pkgname, version and revision are from common/shlibs, so if you change the version there, xbps-src will generate a dependency on libvterm>=0.3_1.

Reply:
Ah, good to know.